evalml/automl/automl_search.py

```diff
@@ -284,12 +285,19 @@ def __init__(self,
         self.search_iteration_plot = None
         self._interrupted = False
 
+        if len(self.problem_configuration):
+            pipeline_params = {**{'pipeline': self.problem_configuration}, **self.pipeline_parameters}
+        else:
+            pipeline_params = self.pipeline_parameters
+
         if self.allowed_pipelines is None:
             logger.info("Generating pipelines to search over...")
             allowed_estimators = get_estimators(self.problem_type, self.allowed_model_families)
             logger.debug(f"allowed_estimators set to {[estimator.name for estimator in allowed_estimators]}")
-            self.allowed_pipelines = [make_pipeline(self.X_train, self.y_train, estimator, self.problem_type, custom_hyperparameters=self.pipeline_parameters) for estimator in allowed_estimators]
-
+            if is_time_series(self.problem_type):
+                self.allowed_pipelines = [make_pipeline(self.X_train, self.y_train, estimator, self.problem_type, pipeline_params, custom_hyperparameters=pipeline_params) for estimator in allowed_estimators]
+            else:
+                self.allowed_pipelines = [make_pipeline(self.X_train, self.y_train, estimator, self.problem_type, None, custom_hyperparameters=pipeline_params) for estimator in allowed_estimators]
```

Comment:
FWIW, trying to separate out `pipeline_params` from custom_hyperparameters (though still passing to custom_hyperparameters since this is what's expected), but can't say I've throughly tested this 😓  just used this to pass parameters to make_pipeline for time series pipelines

@jeremyliweishih @freddyaboulton @dsherry

Reply:
Instead of if/else here based on `is_time_series` can we just do this
```python
self.allowed_pipelines = [make_pipeline(self.X_train, self.y_train, estimator, self.problem_type, parameters=parameters, custom_hyperparameters=self.pipeline_parameters) for estimator in allowed_estimators]
```
?

(RE another comment I left above, I renamed your variable `pipeline_params` to `parameters` in my example here)

I don't think the tuner needs to have access to the pipeline-level parameters, right? In other words, saying `custom_hyperparameters=self.pipeline_parameters` here like I did is fine.